### PR TITLE
Fixing broken links and references

### PR DESCRIPTION
**Pull request by matzew**
Updated some links.

@danielpassos @lholmquist can you review/merge?

### CODE REVIEW

File: docs/unifiedpush/ups_userguide/next.asciidoc [hunk 1]
```diff
@@ -37,15 +35,15 @@ For the backend integration of your server we have a few different tutorials as
 
 === Quickstarts
 
-* link:https://github.com/aerogear/aerogear-push-helloworld/releases/latest[HelloWorld for Android, Cordova and iOS]
-* link:https://github.com/aerogear/aerogear-push-quickstarts/releases/latest[MobileContacts for Android, Cordova and iOS + JAX-RS and Fabric8]
-* link:https://github.com/aerogear/aerogear-simplepush-unifiedpush-quickstart[SimplePush and UnifiedPush]
+* link:https://github.com/jboss-mobile/unified-push-helloworld/releases/latest[HelloWorld for Android, Cordova and iOS]
+* link:https://github.com/jboss-mobile/unified-push-quickstarts/releases/latest[MobileContacts for Android, Cordova and iOS + JAX-RS and Fabric8]
```
Comment (lholmquist) [inline docs/unifiedpush/ups_userguide/next.asciidoc]: Should these point to the "release" page or the readme ?

**Comment by lholmquist** [comment]
other than that one comment :+1:

**Comment by danielpassos** [comment]
lgtm

**Comment by matzew** [comment]
@lholmquist good catch :-) fixed it

**Pull request merged**